README.md

```diff
@@ -84,27 +84,36 @@ These control the size of the viewport in your app.
 
 ##### `imageHeight` & `imageWidth` (Number)
 
-The height and width of the image you wish to render.
+The height and width of the image you wish to render. They are not necessary
+if you use tiff.
 
 ##### `initialViewState` (object) [ONLY NECESSARY FOR `MicrsocopyViewer`]
 
 An object containing two things
+
 - `target` (Array) An `[x,y,0]` location in image coordinates of the image. The 0
   represents a hypothetical (and potentially future addition) of a third spatial dimension.
 - `zoom` (Number) The initial zoom level to render the image at
 
 ##### `minZoom` & `maxZoom` (Number)
 
 These control the max and min zoom sizes, generally the number of images `n` in your pyramid,
-ranging from `-n` (zoomed out) to `0`, the highest resolution.
+ranging from `-n` (zoomed out) to `0`, the highest resolution. They are not necessary
+if you use tiffs.
 
-##### `sliderValues` (Array) **POTENIAL FUTURE BREAKING CHANGES WITH NEW FEATURES**
+##### `sliderValues` (Array)
 
 An object containing slider (max/min) values for each channel:
-`{sliderValues:{name:value}, {name:value}, {name:value}}`
+`{sliderValues:{name:value, name:value, name:value}}`
```

Comment:
Couple you provide an actual sample? It confuses me right now: 
- If sliderValues is an array, the example should be an array.
- "sliderValues" is a key that would actually be in the output, but each "name" should be different, right?
- How are min and max encoded here?

Reply:
Replace `value` with `[max, min]`... I think that's correct?